platform/android/SCsub

```diff
@@ -23,8 +23,6 @@ android_files = [
 ]
 
 env_android = env.Clone()
-if env['target'] == "profile":
```

Comment:
That was legacy stuff, we don't have this target anymore and this define doesn't match any current code.